releasenotes/notes/add-fit-model-and-remove-curve_fitter-arg-4a0effb5f9b88ba9.yaml

```diff
@@ -0,0 +1,24 @@
+---
+features:
+  - |
+    :attr:`CurveAnalysis.fit_model` and :attr:`CurveAnalysis.parameters` have been
+    added to the curve analysis and its subclasses. These properties return
+    :class:`FitModel` instance that represents a fit function of the analysis
+    and parameters to be fit with the analysis, respectively.
+  - |
+    New classes :class:`SingleFitFunction` and :class:`CompositeFitFunction` have been
+    added to the curve analysis module. These classes abstract the curve definition
+    hard-coded in the class attribute :attr:`CurveAnalysis.__series__` as a
+    Python function-like object. This may help user to grasp the fitting model
+    without reading the code.
+upgrade:
+  - |
+    Analysis option `curve_fitter` of the :class:`CurveAnalysis` has be removed
+    because of the serialization problem. To use custom curve fitting library,
```

Comment:
Which serialization problem?

Reply:
You know, the callable is not serializable.

Reply:
This is not quite true. Please check the update
https://github.com/Qiskit/qiskit-experiments/pull/726/commits/cd19b811ad5a98d8d4fe7549ffa1ad6b4daa9df5